jetcd-core/src/test/java/com/coreos/jetcd/internal/impl/KVTest.java

```diff
@@ -221,4 +227,9 @@ public void testNestedTxn() throws Exception {
     test.assertEquals(getResp2.getKvs().size(), 1);
     test.assertEquals(getResp2.getKvs().get(0).getValue().toStringUtf8(), oneTwoThree.toStringUtf8());
   }
+
+  @AfterTest
+  public void stop() throws IOException {
```

Comment:
maybe use `public void tearDown()` instead? that is being used in everywhere else i think.